charts/bulk-scan-payment-processor/values.yaml

```diff
@@ -3,3 +3,17 @@ java:
   ingressHost: bulk-scan-payment-processor-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal
   environment:
   image: 'https://hmctspublic.azurecr.io/bulk-scan/payment-processor:latest'
+  PAYMENTS_QUEUE_CONNECTION_STRING: "$(SB_CONN_STRING);EntityPath=payments"
+  PAYMENTS_QUEUE_MAX_DELIVERY_COUNT: "10"
```

Comment:
We need to make sure this value is slightly smaller than the queue's actual max delivery count. We did that in the orchestrator in order to make sure it will always be the application dead-lettering the message, not the queue itself. Otherwise, there would be a chance that we wouldn't get the related alert, in case some random failure prevented the app from dead-lettering.

Reply:
for now change the count 5